docs/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,39 @@
+# Code of Conduct 
+(impending updates on the horizon)
+
+We believe that anyone attending a Code for America event or participating in our online community should feel safe and be free from harassment.
```

Comment:
I was under the impression that we are no longer part of code for america. Maybe pass this by Hugh and see what the best thing to call ourselves is?

Reply:
I think we should be fine using the code of conduct itself, but just replace 'Code for America' with whatever our name is.

Reply:
wasn't sure if it was transitional or not still. I will follow up and edit/update as appropriate but this is what was available and just copy/pasted from the current code for pdx website.

Reply:
I asked Hugh, he said to just link to the code of conduct that is pinned to the code for pdx github (which this was just copy and pasted from) so the reference is fine until it is actually updated.